org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -24,6 +25,8 @@
   "devDependencies": {
     "optimist": "*",
     "ava": "^0.15.2",
-    "es6-sleep": "*"
+    "es6-sleep": "*",
+    "opent2t-device-smartthingshub": "file:../../../org.opent2t.sample.hub.superpopular/com.smartthings.hub/test",
+    "opent2t-device-binaryswitch": "file:../../test"
```

Comment:
that's actually cool that you can use this syntax to tell npm to pull files locally;  Is there a way to tell it to pull it locally + from teh server (fallback to local incase server fails) or something?